test/verify/check-journal

```diff
@@ -475,36 +479,65 @@ s.send(b"PRIORITY=3\\nFOO=bar\\n")'
         sel = "#journal-entry-fields .nav .btn-danger"
         b.wait_visible(sel)
 
-        sel = "#journal-entry-fields .nav .btn-primary:contains('Report')"
+        sel = "#journal-entry-fields .nav .problem-btn > .btn:contains('Report')"
+        b.click(sel)
+
+        sel = ".modal .workflow-table a:contains('Report to ABRT Analytics')"
+        b.click(sel)
+
+        sel = ".modal .modal-footer .btn:contains('Back')"
+        b.wait_visible(sel)
```

Comment:
To be fair, I don’t remember if I actually updated the tests to work with the current changes, so this might be left over from the previous iteration.